node/node.go

```diff
@@ -222,6 +228,16 @@ func (n *Node) StartConsensus() error {
 		return err
 	}
 
+	if n.appStatusHandler != nil {
+		bootstrapper.AddSyncStateListener(func(b bool) {
+			if b {
+				n.appStatusHandler.SetUInt64Value(core.MetricIsSyncing, uint64(0))
```

Comment:
you could have set a variable here and then you could have had only one call to n.appStatusHandler.SetUInt64Value

Reply:
done